src/services/StatisticsService.js

```diff
@@ -0,0 +1,22 @@
+import instance from '../utilities/axios/axios';
+
+export async function getStatistics() {
+  try {
+    const response = await instance.get('/stats/total');
+
+    if (response?.data?.errors) {
+      throw new Error(
+        `Error getting stats: ${JSON.stringify(response.data.errors)}`,
+      );
+    }
+
+    return response?.data?.data ?? null;
+  } catch (error) {
+    if (instance.isAxiosError(error)) {
```

Comment:
Where did you find this method? I could not find any documentation for it

Reply:
axios.isAxiosError is a built-in method of the Axios library that is used to check whether the error caught in the catch block is an instance of an Axios error. It is defined in the Axios source code. Many examples of isAxiosError usage can be found on resources like Stack Overflow or GitHub, where developers discuss this method. 
I don't remember exactly where I found it, but for example: https://stackoverflow.com/questions/74078008/axiosinstance-and-isaxioserror
https://github.com/axios/axios/blob/v1.4.0/dist/axios.js#L2968